packages/ava-react/zh-CN/README.zh-CN.md

```diff
@@ -26,12 +34,33 @@ $ npm install @antv/ava-react --save
 $ yarn add @antv/ava-react
 ```
 
-## 用法
+下面我们结合实际案例了解 [@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react) 三个核心组件的使用方式：
 
-```ts
-import { NarrativeTextVis } from '@antv/ava-react';
-```
+1. \<NarrativeTextVis/> 使用示例：
```

Comment:
Replace `\<NarrativeTextVis/>` with `` `<NarrativeTextVis/>` `` would be better.

![image](https://github.com/antvis/AVA/assets/15177000/ee87ff95-20d1-4bbf-a8f5-544a42990efb)

Reply:
done~